android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/DownloadResultReceiver.java

```diff
@@ -47,6 +47,12 @@ protected void onReceiveResult(int resultCode, Bundle resultData) {
         packageIntent.putExtras(bundle);
         context.startActivity(packageIntent);
         break;
+      case FileUtils.DOWNLOAD_CANCELLED :
+        downloadedFilename = resultData.getString("filename");
+        String message = String.format(context.getString(R.string.cancelled_downloading_package), downloadedFilename);
+        Toast.makeText(context, message,
+          Toast.LENGTH_SHORT).show();
+        break;
```

Comment:
I wonder if we really need the toast to tell the user that the action they just cancelled was cancelled 😬

Reply:
> so long as there is a way to signal that the user is no longer interested in the results of the download

Oh, I thought you meant signal to the user. 🤷‍♂️ OK, I can make that a no-op